packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,310 @@ components:
             created
           example: 1587667174725
 
+    asset:
+      type: object
+      table: asset
+      additionalProperties: false
+      required:
+        - name
+      properties:
```

Comment:
I think we are missing a `size` property on the root of the asset object as well.